models/validation-inference-scripts/root-cause-models/root-cause-inference.py

```diff
@@ -0,0 +1,119 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Example Usage:
+python \
+root-cause-inference.py \
+    --validationdata ../../datasets/validation-data\
+    /root-cause-validation-data.jsonlines \
+    --model ../../root-cause-models/rootcause-binary-bert-20221118.onnx \
+    --vocab ../../training-tuning-scripts/rootcause-models\
+    /resources/bert-base-uncased-hash.txt \
```

Comment:
not a valid `vocab` path. `resources` directory does not exist. should probably point to `morpheus/data/bert-base-uncased-hash.txt`

Reply:
I created that path now added the vocab file

Reply:
which seems to be the same way as phishing.